src/components/AccessibilityBody/index.jsx

```diff
@@ -35,12 +36,12 @@ const AccessibilityBody = (props) => {
         <WrappedMessage
           message={messages.a11yBodyEmailHeading}
           values={{
-            emailElement,
+            emailElement: ReactDomServer.renderToString(emailElement),
           }}
         >
           {
             displayText => (
-              <li>{displayText}
+              <li><span dangerouslySetInnerHTML={{ __html: displayText }} />
```

Comment:
This is a workaround described in : https://stackoverflow.com/questions/59542379/react-intl-wrapping-child-elements-in-formatmessage?noredirect=1#comment105255145_59542379


If you are certain about the code you want to insert is **not** malicious it's a pretty safe bet to use `dangerouslySetInnerHTML`. If you trust the API you have little to worry about. For example, this is perfectly safe:
```
return <div dangerouslySetInnerHTML={{__html: "<p>foo bar</p>"}} />;
```

Reply:
@awaisdar001 Why is this necessary? Why could we not leave this the way it was before?

Reply:
@fysheets Currently, we are displaying information correctly on the studio-a11y page.
We are supposed to display 


> Notify us by email addressed to [accessibility@edx.org](mailto:accessibility@edx.org).  In order for edX to fully and promptly assess and respond to your request, please provide the following information:


but due to this bug, we instead display (See: https://studio.edx.org/accessibility)

> Notify us by email addressed to


This is why I have to take this workaround which is an issue in `react-intl`. 


if we leave this the way it was before it will be rendered as 

> Send an email to

where it should 

> Send an email to [accessibility@edx.org](ddd) with the following information:

I have spent the time to make it work which looks elegant in the backend, but in the end, the only workaround I had was this. 
Here is the sandbox: 
https://codesandbox.io/s/formattedmessage-render-props-c6hfn

Reply:
@awaisdar001 As discussed in person, thank you for the clarity can you please add a comment to this file to briefly explain this decision so that future editors don't have to find this PR for context.